src/Components/Footer/Footer.scss

```
@@ -0,0 +1,64 @@
+$border: 0.1rem solid #e8ebed;
+
+.Footer {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin-top: 30px;
+  padding-top: 30px;
+  border-top: $border;
+
+  .footerTop {
+    display: flex;
+    flex-direction: row;
+
+    .csCenter {
+      width: 50%;
+      p:first-child {
+        font-weight: 700;
+        margin-bottom: 15px;
+      }
+      div {
+        span:first-child {
+          font-weight: 600;
+        }
+      }
+    }
+
+    .companyinfo {
+      width: 25%;
+      margin-left: 20px;
+      p:first-child {
+        margin-bottom: 10px;
+        font-weight: 600;
+      }
+      p {
+        margin-bottom: 5px;
+        font-size: 14px;
+      }
+    }
+
+    .followus {
+      margin-left: 20px;
+      p {
```

css 선택자 간에는 가독성을 위해 한 줄 띄어주시기 바랍니다. 
```suggestion
      margin-left: 20px;
      
      p {
```

---

넵 잘알겠습니다~